src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,328 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    static Task[] taskList = new Task[100];
+    static int taskCount=0;
+
+    // method prints a horizontal line.
+    public static void displayLine(){
+        System.out.println("____________________________________________________________");
+    }
+
+    // method prints Duke greeting.
+    public static void greeting(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("You are entering the\n" + logo + "\nZone...\n");
+
+        displayLine();
+        System.out.println("Hey there! Duke here!");
+        System.out.println("How can I serve you today?");
+        displayLine();
+    }
+
+    // method prints Duke termination.
+    public static void goodbye(){
+        System.out.println("Goodbye. See you in the funny papers.");
+        displayLine();
+    }
+
+
+    // method prints taskList.
+    public static void printList(){
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i++){
```

Comment:
According to the Java coding standard, the for loop should resemble this (take note of the white space):
```
for (i = 0; i < 100; i++) {
    sum += value[i];
}
```